source/internal-communication.rst

```diff
@@ -481,7 +482,7 @@ History
 
 History is also a very simple service: whenever a user or application sends a
 request to it, it will query MongoDB and build a proper message to send back
-to the user/application. This is shown in :numref:`History`.
+to the user/application. This is shown in :numref:`History` (ver se link funciona).
```

Comment:
I think this kind of comment should be properly commented as the language defines it: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments